appeals/web/src/server/lib/nunjucks-template-builders/tag-builders.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @typedef HtmlLink
+ * @type {object}
+ * @property {string} title
+ * @property {string} href
+ * @property {string?} target
+ */
+
+/**
+ * @param {HtmlLink} link
+ * @returns {string}
+ */
+export const buildHtmlLink = (link) => {
+	return [
+		`<a href="${link.href}"`,
+		link.target ? ` target="${link.target}"` : '',
+		' class="govuk-link">',
+		link.title,
+		'</a>'
+	].join('');
+};
+
+/**
+ * @param {string[]} items
+ * @returns {string}
+ */
+export const buildHtmUnorderedList = (items) => {
+	const listItems = items.map((i) => `<li>${i}</li>`).join('');
+	return `<ul class="govuk-!-margin-top-0 govuk-!-padding-left-0">${listItems}</ul>`;
```

Comment:
This `<ul>` tag should probably also have the `govuk-list` class.

Ref: https://design-system.service.gov.uk/styles/typography/#lists

Reply:
I'll add, lifted from Alex's old template